sematic/types/types/link.py

```diff
@@ -0,0 +1,17 @@
+# Standard library
+from dataclasses import dataclass
+from urllib.parse import urlparse
+
+
+@dataclass
+class Link:
```

Comment:
We should have some documentation on this class pointing out that:
- this represents a URL with label text to replace the URL in visualizations
- the URL must include the scheme
- there is a custom visualization when this type is used that will render it as a button on the UI